javascripts/services/contactService.js

```diff
@@ -30,7 +30,33 @@ app.service("contactService", function($http, $q, FIREBASE_CONFIG){
         return $http.delete(`${FIREBASE_CONFIG.databaseURL}/contacts/${contactId}.json`);
     };
 
-    return {addNewContact, getContacts, deleteContact};
+    const favoriteContact = (contactId) => {
```

Comment:
this isn't right.  you need to be sending something to put.  firebase isn't updating